setup.py

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/python3
+# -*- coding: utf-8 -*-
+
+"""Setup script."""
+
+# Copyright (c) 2022 SUSE LLC
+#
+
+from setuptools import find_packages, setup
+
+with open('README.md') as readme_file:
+    readme = readme_file.read()
+
+with open('requirements.txt') as req_file:
+    requirements = req_file.read().splitlines()
+
+with open('requirements-test.txt') as req_file:
+    test_requirements = req_file.read().splitlines()[2:]
+
+with open('requirements-dev.txt') as req_file:
+    dev_requirements = test_requirements + req_file.read().splitlines()[2:]
+
+tox_requirements = [
+    'tox',
+    'tox-pyenv'
+]
+
+
+setup(
+    name='imagenameparser',
+    version='1.0.0',
+    description="Parse the susecloudpublic image",
+    long_description=readme,
+    long_description_content_type="text/markdown",
+    author="SUSE",
+    author_email='public-cloud-dev@susecloud.net',
+    url='https://github.com/SUSE-Enceladus/suse-cloud-image-name-parser',
+    packages=find_packages(),
+    package_dir={
+        'imagenameparser': 'imagenameparser'
+    },
+    include_package_data=True,
+    python_requires='>=3.4',
```

Comment:
This states 3.4 or larger but below 3.4 and 3.5 are missing. Inconsistent version settings.